src-web/reducers/data-mappers/mapApplicationsSingle.js

```diff
@@ -40,12 +40,18 @@ export const mapSingleApplication = application => {
     //for argo we ask for namespace:targetNamespace label:appLabel kind:<comma separated string of resource kind>
     //this code moves all these items under the related section
     const kind = _.get(item, 'kind')
+    const cluster = _.get(item, 'cluster')
 
-    if (kind === 'application' || kind === 'subscription') {
+    if (kind === 'application') {
       //this is a legit app object , just leave it
       return
     }
 
+    if (kind === 'subscription' && cluster !== 'local-cluster') {
```

Comment:
Maggie, this code is shared by the argo apps, make sure the changes work for both

Reply:
I am not sure why you need to check for local host

Reply:
Thanks @birsanv I'll test argo apps. This change is to avoid showing duplicate sections for remote cluster. So for local cluster, we would have -subscription-1 and -local. While for other remote clusters, the parent is the same as the child

Reply:
The testcase for argo is an argo app deploying acm applications and subscriptions and they can be deployed on local or remote clusters